Mergin/project_dialog.py

```diff
@@ -0,0 +1,42 @@
+import os
+from qgis.PyQt.QtWidgets import QDialog, QDialogButtonBox, QFileDialog, QApplication, QMessageBox
+from qgis.PyQt import uic
+
+ui_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'ui', 'ui_create_project.ui')
+
+
+class ProjectDialog(QDialog):
+    def __init__(self, title, username, user_organisations=None):
+        QDialog.__init__(self)
+        self.ui = uic.loadUi(ui_file, self)
+        self.ui.buttonBox.button(QDialogButtonBox.Ok).setEnabled(False)
+        self.ui.buttonBox.accepted.connect(self.accept_dialog)
+        self.ui.projectNamescape.addItem(username)
+        if user_organisations:
+            self.ui.projectNamescape.addItems([o for o in user_organisations
+                                               if user_organisations[o] in ['admin', 'owner', 'writer']])
```

Comment:
Actually 'writer' is not enough (that's just writing of already existing projects, but does not allow creating/deleting them)